exercises/rest-api/canonical-data.json

```
@@ -1,6 +1,6 @@
 {
     "exercise": "rest-api",
-    "version": "1.0.2",
+    "version": "1.0.3",
```

```suggestion
    "version": "1.1.0",
```

I agree with @petertseng's assessment.

---

Same here.